server/src/test/scala/cromwell/engine/workflow/WorkflowActorSpec.scala

```diff
@@ -328,6 +335,17 @@ class WorkflowActorSpec extends CromwellTestKitWordSpec with WorkflowDescriptorB
         val _ = createWorkflowActor(WorkflowSucceededState, Option(new ThrowingPathBuilderFactory()))
       }
     }
+
+    "send a workflow callback message" in {
+      val actor = createWorkflowActor(FinalizingWorkflowState, workflowCallbackActor = Option(workflowCallbackProbe.ref))
+      deathwatch watch actor
+      val msg = WorkflowCallbackActor.PerformCallbackCommand(currentWorkflowId, Some(mockUri), WorkflowSucceeded, CallOutputs.empty, List.empty)
```

Comment:
1. Is it possible to have a small set of outputs here to validate?
2. Is it possible to have an equivalent test for a failing workflow?